.github/workflows/ci.yaml

```diff
@@ -157,7 +157,7 @@ jobs:
     # TODO: Find some way to automatically upgrade to "latest"
     with:
       do-push: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
-      go-repo-ref: ${{ needs.latest-sdk-versions.outputs.go_latest }}
+      go-ver: 'v${{ needs.latest-sdk-versions.outputs.go_latest }}'
```

Comment:
I thought you wanted to remove this `v` too.

Reply:
Nope, the all-docker-images.yaml workflow expects (and requires) the `v`: https://github.com/temporalio/features/blob/main/.github/workflows/all-docker-images.yaml#L7-L8
(it looks like it doesn't actually in practice, it might work without, but it does work with, and I'd rather keep things consistent)